// Copyright 2018 The LUCI Authors.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

// COPY of go.chromium.org/luci/buildbucket/proto/launcher.proto

syntax = "proto3";

package swarming.bb;

// This file is meant for protos passed and used internally.

// A collection of build-related secrets we might pass from Buildbucket to Kitchen.
message BuildSecrets {
  // Token to identify RPCs associated with the same build.
  string build_token = 1;

  // Token to allow updating this build's invocation in ResultDB.
  string resultdb_invocation_update_token = 2;
}